pipelinewise/fastsync/mysql_to_snowflake.py

```diff
@@ -25,8 +25,6 @@
         'user',
         'password',
         'warehouse',
-        'aws_access_key_id',
-        'aws_secret_access_key',
```

Comment:
can you please make the same change in [postgres_to_snowflake](https://github.com/transferwise/pipelinewise/blob/master/pipelinewise/fastsync/postgres_to_snowflake.py) and [s3_csv_to_snowflake](https://github.com/transferwise/pipelinewise/blob/master/pipelinewise/fastsync/s3_csv_to_snowflake.py)

Reply:
Sure, Removed it in postgres and s3_csv as well

Reply:
thanks